recipe/meta.yaml

```diff
@@ -10,19 +10,19 @@ source:
   sha256: e2714f07b750458beaf5b07b670ea7b6058ee155c021c987d0b8e6a40bf3446f
 
 build:
-  noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
   run:
-    - python >=3.7
+    - python
     - grpcio >=1.38.1,<2.0dev
     - google-api-core-grpc >=1.34.0,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
-    - proto-plus >=1.22.0,<2.0.0dev
+    - proto-plus >=1.22.0,<2.0.0dev  # [py<311]
+    - proto-plus >=1.22.2,<2.0.0dev  # [py>=311]
```

Comment:
Did I get this right?

Reply:
Based on my read of [docs], yes. I think your expertise here exceeds mine, though.

[docs]: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors

Reply:
Is this minute difference enough to warrant an arch dependent package?

Can we just leave the package no arch